### ISSUE TITLE
Water Candle breaking blocks

### ISSUE THREAD

**Issue by ShayBox** [Lothrazar/Cyclic#1041]
Water candles are breaking blocks when spreading, which keep growing indefinitely like a virus. To figure this out I had to use tickrate changer mod, not a problem between the two mods, it just speeds up the spreading process enough to notice it.

**Comment by Lothrazar** [comment]
Does it break when the mob spawns in, in the same place as the mob? or just at random unrelated times ?

**Comment by moneyblind** [comment]
I can confirm this, it has voided blocks in the vicinity. It appears to simply replace the block (it appears to be matching the current y level. on all the spawns)

To replicate this (on latest release to date):

I placed the candle in the foreground of the screenshot.
I used the Acceleration Wand to force the candle to display its mechanics (otherwise i could not catch in the act.)
It randomly places mobs (another photo attached showing the mobs spawning in the walls at the outermost range of the watercandle).
It does not always replace blocks with a candle, it is random.
I have found candles embedded in the wall.

Of interesting note. As the Initial issue was reported, the candles seem to "infect" and use any candles location as a radius center. As evident in the candle photo, the candle furthest from the shot is out of range of the first candle, but not the second.

This is resulting of a complete infestation which voids blocks not limited to stone. (I lost a mob crusher which made me do these tests).

<img width="1920" alt="1 8 0_51 screenshot 2019 02 03 - 02 34 24 97" src="https://user-images.githubusercontent.com/7253787/52174636-77afdf80-275c-11e9-9a70-0daf74484e23.png">

<img width="1920" alt="1 8 0_51 screenshot 2019 02 03 - 02 32 41 13" src="https://user-images.githubusercontent.com/7253787/52174727-8c40a780-275d-11e9-8275-8b7cef6dcefa.png">
